analysis/src/SharedTypes.ml

```diff
@@ -425,6 +425,12 @@ module Completable = struct
     | CPObj of contextPath * string
     | CPPipe of contextPath * string
 
+  type typedContext = NamedArg of string
+
+  let typedContextToString typedContext =
+    match typedContext with
+    | NamedArg argName -> "NamedArg(" ^ argName ^ ")"
+
   type t =
     | Cdecorator of string  (** e.g. @module *)
     | CnamedArg of contextPath * string * string list
```

Comment:
`~prefix` with `[this, that]` seen.
So it would complete a label whose name starts with `prefix` and excludes labels `this` and `that` which have been used already and should not be suggested.

Reply:
I've done that now, _but_ I haven't done anything about `this`/`that` seen, because I don't think it makes sense in the context of completing a labelled argument? There's nothing else to have been seen already, right?